source/ShipInfoPanel.cpp

```diff
@@ -507,12 +507,15 @@ void ShipInfoPanel::DrawWeapons(const Rectangle &bounds)
 	auto layout = Layout(static_cast<int>(LABEL_WIDTH), Truncate::BACK);
 	for(const Hardpoint &hardpoint : ship.Weapons())
 	{
-		string name = "[empty]";
+		
+		bool isRight = (hardpoint.GetPoint().X() >= 0.);
+		bool isTurret = hardpoint.IsTurret();
+		string name = "[empty510]";
+		if(isTurret == true)
+			name = "[empty turret mount]";
```

Comment:
Would it be possible to do somethig like this?
```suggestion
			name = "[empty turret mount]";
		bool isGun = hardpoint.IsGun();
		if(isGun == true)
			name = "[empty gun port]";
		bool isPylon = hardpoint.IsPylon();
		if(isPylon == true)
			name = "[empty pylon]";
```

Reply:
Might be, although the `isGun" stuff is wonky because the game currently doesn't really identify guns, it mostly just identifies things that aren't turrets or pylons. That's something I want to work on fixing here. This first label is just the starting point.